main.py

```diff
@@ -2,6 +2,54 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+
+    # Create
+    genres = ["Western", "Action", "Dramma"]
+    for genre_name in genres:
+        Genre.objects.create(name=genre_name)
+
+    names = [
```

Comment:
isn't it better to rename to `actors`? What do you think?
Probably not all of names will be actors - that what developer could think when look at this
Try to clarily